console/billing/how-to/add-payment-method.mdx

```diff
@@ -0,0 +1,68 @@
+---
+meta:
+  title: How to add a payment method
+  description: This page explains how to add a payment method
+content:
+  h1: How to add a payment method
+  paragraph: This page explains how to add a payment method
+tags: billing payment sepa credit-card
+dates:
+  validation: 2024-02-16
+  posted: 2024-02-16
+categories:
+  - billing
+---
+
+
+
+Before you can order Scaleway resources, you must add your payment method to your account.
+
+<Macro id="iam-requirements" />
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```

Comment:
Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.